testdata/go/v3/memcached-operator/PROJECT

```diff
@@ -1,8 +1,13 @@
 domain: example.com
 layout: go.kubebuilder.io/v3
 plugins:
-  manifests.sdk.operatorframework.io/v2: {}
-  scorecard.sdk.operatorframework.io/v2: {}
+  manifests.sdk.operatorframework.io/v1:
```

Comment:
That is terrific and very nice 👍 
We can have the data now whcih would indeed allow us in the future tooling the migration from go/v3 to go/v4 as if we wish to create from that a structure required for the config-gen as well. 

I am assuming that the projects which have the plugins scaffolded already without the data used to config them will still be working. Am I right? If yes it has my 1000% support. 

c/c @estroz @jmrodri @joelanford

Reply:
They work. They may have errors if going the other way (scaffolding with a new version and then using and old version afterwards) but I'm not 100% sure about that.

Reply:
It is solved now. thank you.